openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -1507,7 +1507,96 @@ It is up to the Credential Issuer whether to update both the signature and the c
 
 # Privacy Considerations
 
-TBD
+When [@!RFC9396] is used, the Privacy Considerations of that specification also apply.
+
+The privacy principles of [@ISO.29100] should be adhered to.
+
+## User Consent
+
+The Credential Issuer should obtain the End-User's consent before issuing Credential(s)
+to the Wallet. It should be made clear to the End-User what information is being included in the
+Credential(s) and for what purpose.
+
+## Minimum Disclosure
+
+To ensure minimum disclosure and prevent Verifiers from obtaining claims unnecessary for the
+transaction at hand, when issuing Credentials that are intended to be created once and then used a
+number of times by the End-User, the Credential Issuers and the Wallets SHOULD implement
+Credential formats that support selective disclosure, or consider issuing a separate Credential
+for each user claim.
+
+## Storage of the Credentials
+
+To prevent a leak of End-User data, especially when it is signed, which risks revealing private data of End-Users to
+third parties, systems implementing this specification SHOULD be designed to minimize the amount
+of End-User data that is stored. All involved parties SHOULD store Verifiable Credentials
+containing privacy-sensitive data only for as long as needed, including in log files. Any logging of
+End-User data should be carefully considered as to whether it is necessary at all. The time logs are retained
+for should be minimized.
+
+After Issuance, Credential Issuers SHOULD NOT store the Issuer-signed Credentials if they
+contain privacy-sensitive data. Wallets SHOULD store Credentials only in encrypted form, and,
+wherever possible, use hardware-backed encryption. Credentials that are no longer valid to present should be deleted as soon as
+possible.
+
+## Correlation 
+
+### Unique Values Encoded in the Credential
+
+Issuance/presentation or two presentation sessions by the same End-User can be linked on the basis of
+unique values encoded in the Credential (End-User claims, identifiers, Issuer signature, etc.) either by colluding Issuer/Verifier or Verifier/Verifier pairs, or by the same Verifier.
+
+To prevent these types of correlation, Credential Issuers and Wallets SHOULD use
+methods, including but not limited to the following ones:
+
+* Issue a batch of Credentials to enable the usage a unique Credential per presentation or per Verifier using Batch Credential Endpoint defined in (#batch-credential-endpoint). This only helps with Verifier/Verifier unlinkability.
+* Use cryptographic schemes that can provide non-correlation.
+
+### Credential Offer
+
+The Privacy Considerations in Section 11.2 of [@!RFC9101] apply to the `credential_offer` and
+`credential_offer_uri` parameters defined in (#credential-offer).
+
+### Authorization Request
+
+The Wallet SHOULD NOT include potentially sensitive information in the Authorization Request,
+for example, that including clear-text session information as a `state` parameter value, or encoding
```

Comment:
This sentence doesn't quite read right to me. Here's an attempt to fix while keeping what I think was the intent:

```suggestion
for example, by including clear-text session information as a `state` parameter value or encoding
```